src/async_impl/client.rs

```diff
@@ -99,6 +99,10 @@ struct Config {
     #[cfg(feature = "__tls")]
     tls_built_in_root_certs: bool,
     #[cfg(feature = "__tls")]
+    min_tls_version: Option<Option<TlsVersion>>,
```

Comment:
I think this is supposed to be just 1 option.

Reply:
The idea is:
- `None` means do nothing, follow the backend's own default
- `Some(None)` means remove the minimum, allow any version
- `Some(Some(version))` means set your own minimum

That means that if `native-tls` changes its default minimum `reqwest` will automatically follow along.

(`max_tls_version` is simpler because no limit is a sane default.)

Reply:
Oh, okay. If sticking with that, the 3 different states, perhaps a specific enum would make that clearer. But perhaps there isn't a need to remove a minimum. If a user doesn't want the default (backend default), then perhaps they should have to specify a minimum.